doc/userguide/rules/intro.rst

```diff
@@ -280,6 +280,35 @@ keywords.
 
 Some generic details about keywords follow.
 
+Disabling Alerts
+~~~~~~~~~~~~~~~~
+There is a way to disable alert generation for a rule using the keyword ``noalert``.
+When this keyword is part of a rule, no alert is generated if the other
+portions of the rule match. That is, the other rule actions will *still be
+applied.* Using ``noalert`` can be helpful when a rule is
+collecting or setting state using `flowbits`, `datasets` or other
+state maintenance constructs of the rule language.
```

Comment:
Suggestion: add a reference to the thresholding section, for those interested in controlling alert frequency: https://docs.suricata.io/en/latest/rules/thresholding.html#thresholding-keywords

Reply:
Good point.